python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -233,6 +232,13 @@ class LoadReferenceObjectsConfig(pexConfig.Config):
         dtype=bool,
         default=False,
     )
+    cal_ref_cat = pexConfig.Field(
```

Comment:
Why is this called `cal_ref_cat`? That's not the name of any config field. The config field name in `LoadIndexedReferenceObjectsConfig` is `ref_dataset_name`, with a default value of `"cal_ref_cat"`. We want no default value here, certainly.

I'm assuming the rationale of this field is to be able to read in previously written configs? Maybe the NCSA->USDF transition will help with that, by pruning out the oldest runs, but we've always had a danger with removing config fields because old runs stick around a long time. I don't know how many cases it might actually come up though, and I think your approach here isn't terrible? Especially as long as this field doesn't get written when it's defaulted (does it?).

Reply:
This was a really dumb typo.

Reply:
I just checked, and defaulted+deprecated fields don't get written, so that's fine.